certification/internal/policy/operator/operator_pkg_name_uniqueness.go

```diff
@@ -174,8 +174,8 @@ func (p *OperatorPkgNameIsUniqueCheck) Metadata() certification.Metadata {
 	return certification.Metadata{
 		Description:      "Validating Bundle image package name uniqueness",
 		Level:            "best",
-		KnowledgeBaseURL: "https://connect.redhat.com/zones/containers/container-certification-policy-guide",
-		CheckURL:         "https://connect.redhat.com/zones/containers/container-certification-policy-guide",
+		KnowledgeBaseURL: "https://sdk.operatorframework.io/docs/olm-integration/tutorial-package-manifests/",
+		CheckURL:         "https://sdk.operatorframework.io/docs/olm-integration/tutorial-package-manifests/",
```

Comment:
Package manifests have been deprecated in favor of bundles. Do we want to send people to that article? I get that there may not be a specific article regarding package naming but I'm not sure that this article is the best alternative.

Reply:
thanks. changed the link to point to bundle tutorial.